.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
Think we'd better add `.vscode` and `.idea` to SDK's `.gitignore` as well. 🙇🏻

Reply:
Done